docs/source/index.rst

```diff
@@ -1,5 +1,8 @@
 .. Koalas' documentation master file
 
+.. image:: ../../Koalas-logo-nolabel.png
+    :width: 160px
```

Comment:
How about placing the logo like pyspark? (https://spark.apache.org/docs/latest/api/python/index.html)
I think it's also fine to just don't put the logo - seems pandas doesn't do that as well

Reply:
I got an advice from our designer.

Reply:
+1 for the current one